set sentry for frontend

Blue-B-code opened the pull request:
This PR complements the ones below by integrating the Sentry SDK with the GlitchTip platform in the React.js frontend.
It allows better tracking and capturing of bugs or errors in your frontend application in an easier and more efficient way.

https://github.com/openimis/openimis-fe_js/pull/202
https://github.com/openimis/openimis-fe-core_js/pull/288

Blue-B-code committed:
set sentry for frontend

Blue-B-code committed:
corrections

Blue-B-code committed:
corrections

Blue-B-code committed:
corrections

Blue-B-code committed:
corrections

Blue-B-code committed:
Merge branch 'develop' into feature-glitchtip

zikani03 reviewed (changes requested):
@Blue-B-code @mngoe please clean up the readme to not add back the content we removed to streamline the instructions, otherwise looks good.

Blue-B-code committed:
Update README with Sentry frontend setup

Added instructions for using Sentry in the frontend.

Blue-B-code commented:
> @Blue-B-code @mngoe please clean up the readme to not add back the content we removed to streamline the instructions, otherwise looks good.

fixed
